Sources/Brave/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -208,6 +213,30 @@ class ShieldsViewController: UIViewController, PopoverContentComponent {
       height: height
     )
   }
+  
+  private lazy var reportViewController: UIHostingController<SubmitReportView>? = {
+    guard let url = url else { return nil }
+    var components = URLComponents(url: url, resolvingAgainstBaseURL: false)
+    components?.fragment = nil
+    components?.queryItems = nil
+    guard let cleanedURL = components?.url else { return nil }
```

Comment:
Why are you not using `domainURL` or one of the many display variants we have in Shared/BraveShared here?

Reply:
domainURL strips the path. I want to strip just fragments and query params nothing else. So not exactly sure what I'm supposed to use.